docs/source/cross_compile/config_vs_code.rst

```diff
@@ -41,26 +41,31 @@ Visual Studio Code, follow the steps below.
    workspace. For example: *<My Documents>\\HelloWorld\\\* or *<My
    Documents>\\NI Linux Real-Time Template\\*.
 
-2. Launch VS Code.
+2. In a new command prompt, navigate to the toolchain folder and run the
+   environment-setup-core2-64-nilrt-linux.bat script.
 
-3. | In VS Code, select **Open folder…** from the landing page or **File
+3. Launch VS Code from the same command prompt, to ensure that the toolchain
+   environment is set up correctly. Hereafter, always launch VS Code for
+   cross-compilation after setting up the environment.
```

Comment:
Is this always necessary even with the config set up in the VS Code environment? I thought this script just added things to the PATH/environment variables. Does it do more than that now?

Reply:
Frankly, I did not try setting up any other way due to instructions [here](https://ni.visualstudio.com/DevCentral/_boards/board/t/RTOS/Work%20Items/?workitem=2105299). The script still does only that, but it seems to add several compiler command-line arguments that is missing from the the VS Code config.

Reply:
I can see a future state where we update the tutorials to run that setup as part of the build process, but for now I think this is fine.